recipes/gmp/all/conanfile.py

```diff
@@ -82,12 +83,11 @@ def build_requirements(self):
             if not self.conf.get("tools.microsoft.bash:path", check_type=str):
                 self.tool_requires("msys2/cci.latest")
         if is_msvc(self):
-            self.tool_requires("yasm/1.3.0")      # Needed for determining 32-bit word size
-            self.tool_requires("automake/1.16.5") # Needed for lib-wrapper
+            self.tool_requires("yasm/1.3.0")
+            self.tool_requires("automake/1.16.5")
```

Comment:
Please put the comments back.

In fact we do not have enough comments and it's desirable some more

Reply:
I can put back yasm eventually, but automake comment is useless since it's a common pattern.